moneo.py

```diff
@@ -63,20 +97,99 @@ def stop(self):
 
     def jobID_update(self):
         '''Updates job id for hosts listed in the specified host ini file'''
-        dep_cmd = 'ansible-playbook' + ' -f ' + str(args.fork_processes) + ' -i ' + \
-            args.host_ini + ' src/ansible/updateJobID.yaml -e job_Id=' + args.job_id
-        print('Job ID update to ' + self.args.job_id)
-        logging.info('Job ID update to ' + args.job_id + ". Hostfile: " + args.host_ini)
-        os.system(dep_cmd)
+        print('Updating job ID to ' + self.args.job_id)
+        cmd = '/tmp/moneo-worker/jobIdUpdate.sh ' + self.args.job_id
+        pssh(cmd=cmd, hosts_file=self.args.host_file)
```

Comment:
Just a general question: How do you handle pssh failures? When using pssh I have found that it's not extremely rare that I will have a node just randomly fail for whatever reason, and if I ran the command again it would pass. Is that a concern?

Reply:
I have added failure checking to the pssh/pscp which throws an exception if there are failures. Would this address that?

Reply:
Yeah, I think that's probably good enough. If it becomes a bigger problem maybe there are other things that could be done, but I guess it's probably best to cross that bridge if it becomes necessary.